.github/workflows/fuzzer.yml

```diff
@@ -68,62 +54,206 @@ jobs:
             ${{github.job}}-${{env.GODOT_BASE_BRANCH}}-${{github.ref}}
             ${{github.job}}-${{env.GODOT_BASE_BRANCH}}
 
-      # Use python 3.x release (works cross platform; best to keep self contained in it's own step)
-      - name: Set up Python 3.x
-        uses: actions/setup-python@v2
+      # NO-PIE is for more detailed bactraces with line symbols
+      - name: Compilation
+        env:
+          SCONS_CACHE: ${{github.workspace}}/.scons_cache/
+        run: |
+          # Helpful to test later code without needing to compile entire editor
+          #wget https://downloads.tuxfamily.org/godotengine/4.0/beta3/Godot_v4.0-beta3_linux.x86_64.zip -O a.zip
```

Comment:
beta4 was released since. This line will have to be updated regularly, not sure if there is an alternative